app/Models/Discord/DiscordRole.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models\Discord;
+
+use Illuminate\Database\Eloquent\Model;
+use Spatie\Permission\Models\Permission;
+
+class DiscordRole extends Model
+{
+    public $timestamps = false;
+
+    public function role()
```

Comment:
Should this be `permission`?

Reply:
Removed in c2d4c3c as the relationship is not used anywhere